cluster/manager/pair.go

```diff
@@ -317,6 +342,41 @@ func (c *ClusterManager) GetPairToken(
 	}, nil
 }
 
+func (c *ClusterManager) generatePairToken() ([]byte, error) {
+	var token []byte
+	var systemSecret string
+
+	if auth.Enabled() {
+		generator, err := auth.Init(&auth.Config{
```

Comment:
Switch this to have `SystemTokenManager`, `SharedSecretTokenGenerator` and then `TokenGenerator.Generate`